toc/working-groups/cf-on-k8s.md

```diff
@@ -61,6 +61,8 @@ areas:
   - name: Tim Downey
     github: tcdowney
   repositories:
+  - cloudfoundry/eirini-controller
```

Comment:
```suggestion
  - cloudfoundry/cf-k8s-secrets
  - cloudfoundry/eirini-controller
```
Let's keep the list in alphabetized order. 😄

Reply:
done!